compilator/compilator.go

```diff
@@ -94,6 +94,7 @@ func NewCompilator(
 		repositoryPrefix: repositoryPrefix,
 		baseType:         baseType,
 		fissileVersion:   fissileVersion,
+		compilePackage:   (*Compilator).compilePackageInDocker,
```

Comment:
Might have been a good place for an interface? This is just an odd mechanism. Works fine though.